.github/workflows/format.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow checks that the code is formatted using rustfmt.
+
+name: tock-ci
+env:
+  TERM: xterm # Makes tput work in actions output
```

Comment:
We don't need this, it was just in the Tock workflow because we use tput in the Makefiles, but it doesn't hurt to leave it here